content/applications/inventory_and_mrp/quality/quality_check_types/instructions_check.rst

```diff
@@ -0,0 +1,86 @@
+and clicking on an order. Inventory orders can be selected by navigating to
+:menuselection:`Inventory`, clicking the :guilabel:`# To Process` button on an operation card, and
+selecting an order.
+
+On the selected manufacturing or inventory order, a purple :guilabel:`Quality Checks` button appears
+above the order. Click the button to open the :guilabel:`Quality Check` pop-up window, from which
+any quality checks created for the order can be processed.
+
+.. image:: instructions_check/quality-check-pop-up.png
+   :align: center
+   :alt: The Quality Check pop-up window on a manufacturing or inventory order.
+
+To complete an Instructions quality check, follow the instructions detailed in the
+:guilabel:`Quality Check` pop-up window. Finally, click :guilabel:`Validate` to confirm that the
+check has been completed.
+
+If a quality alert must be created, click the :guilabel:`Quality Alert` button that appears at the
+top of the manufacturing or inventory order after the check is validated. Clicking
+:guilabel:`Quality Alert` opens a quality alert form on a new page. For a complete guide on how to
+fill out quality alert forms, view the documentation on :ref:`Quality alerts
+<quality/quality_management/quality-alerts>`.
+
+Process a work order quality check
+----------------------------------
+
+When configuring a |QCP| that is triggered by a manufacturing order, a specific work order can also
+be specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a work order is
+specified, an Instructions quality check is created for that specific work order, rather than the
+manufacturing order as a whole.
+
+Quality checks configured for work orders must be completed from the tablet view. To do so, begin by
+navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`. Select a
+manufacturing order that includes a work order for which a quality check is required. Open the
+tablet view for that work order by clicking the :guilabel:`📱 (tablet)` button on the order's line.
+
+With tablet view open, complete the steps listed on the left side of the screen until the
+Instructions quality check step is reached. Upon reaching the check, the instructions for how to
+complete it will appear at the top of the screen. Follow the instructions, then click
+:guilabel:`Next` to move on to the next step.
+
+.. image:: instructions_check/work-order-instructions-check.png
+   :align: center
+   :alt: An instructions check for a work order.
+
+If a quality alert must be created, do so by clicking the :guilabel:`☰ (menu)` button in the tablet
+view, and selecting :guilabel:`Quality Alert` from the :guilabel:`Menu` pop-up window. A
+:guilabel:`Quality Alerts` pop-up window appears, from which a quality alert can be created. For a
+complete guide to quality alert creation, view the documentation on :ref:`Quality alerts
+<quality/quality_management/quality-alerts>`.
```

Comment:
If I remember correctly, the () after the emoji is to describe what the emoji looks like, in case the reader's browser is too old to render the emoji.
```suggestion
If a quality alert must be created, do so by clicking the :guilabel:`☰ (hamburger)` icon in the top
left corner tablet view, and selecting :guilabel:`Quality Alert` from the :guilabel:`Menu` pop-up
window.

Then, a :guilabel:`Quality Alerts` pop-up window appears, from which a quality alert can be
created. For a complete guide to quality alert creation, view the documentation on :ref:`Quality
alerts <quality/quality_management/quality-alerts>`.
```